### PR TITLE
[CI:DOCS] Better document the default value of --userns

### LINKED ISSUE #15764: Unclear default value for --userns
The manual page of podman-run says about `--userns`: “Set the user namespace mode for the container. It defaults to the `PODMAN_USERNS` environment variable.”  I have no PODMAN_USERNS environment variable.

What is the default value for `--userns` (in rootless and rootful mode)?

Podamn Version:      4.3.0-dev

### PR DESCRIPTION
**Pull request by imphil**
Better document which value `podman run --userns` has if no 
value is specified. Also improve documentation of "host" being an alias
for "".

Fixes #15764

#### Does this PR introduce a user-facing change?

None

### CODE REVIEW

**Comment by rhatdan** [comment]
Thanks @imphil 
LGTM
/approve

File: docs/source/markdown/options/userns.container.md [hunk 1]
```diff
@@ -4,19 +4,26 @@
 ####> are applicable to all of those.
 #### **--userns**=*mode*
 
-Set the user namespace mode for the container. It defaults to the **PODMAN_USERNS** environment variable unless `--pod` is specified. An empty value ("") means user namespaces are disabled unless an explicit mapping is set with the **--uidmap** and **--gidmap** options.
+Set the user namespace mode for the container.
+
+If `--userns` is not set the default value is determined as follows.
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/options/userns.container.md:9]:
```suggestion
If `--userns` is not set, the default value is determined as follows.
```

**Comment by imphil** [comment]
Thinking about it a bit more, the configuration file also provides a default value, which is probably between PODMAN_USERNS and the default --userns=host. I'll update the PR with that tomorrow.

**Comment by imphil** [comment]
Thanks for your review, @TomSweeneyRedHat! Updated.

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by rhatdan** [comment]
/lgtm
Thanks @imphil

**Pull request merged**